content/W2017/W17-events-mentorship-cake.md

```diff
@@ -4,7 +4,8 @@ Category: Events
 Tags: social, big csters, 
 Slug: mentoship-night-w17
 Author: Monica Xu
-Summary: Meet other women in CS, connect with upper year CS students, and enjoy some cake!
+Summary: Get upperclassmen help about various CS options, specific upper year classes,
+interviews and other aspects of university life
```

Comment:
For the website to render properly, this needs to be one line. Or else you get this: 
<img width="906" alt="screen shot 2017-02-03 at 12 51 06 pm" src="https://cloud.githubusercontent.com/assets/10576078/22602031/89c4a1ec-ea0f-11e6-9a92-3cb2a5fb7dba.png">